postgrest.dict

```diff
@@ -104,6 +104,7 @@ ORM
 ov
 passphrase
 Pawel
+PBKDF2
```

Comment:
Check the error on https://github.com/PostgREST/postgrest-docs/actions/runs/3481340190/jobs/5822304524

To clear it, maybe try changing this to

```suggestion
PBKDF
```